### PR TITLE
Add a default return in method __str__ from Post Model

### PR DESCRIPTION
**Pull request by rafaelgontijo**

### CODE REVIEW

**Comment by yakky** [comment]
Do you mind targeting `release/0.9.x` branch?

File: djangocms_blog/models.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ class Meta:
         get_latest_by = 'date_published'
 
     def __str__(self):
-        return self.safe_translation_getter('title', any_language=True)
+        return self.safe_translation_getter('title', any_language=True, default="")
```
Comment (yakky) [inline djangocms_blog/models.py]: I'd like more something like `Post (no translation)` (in a gettext marker). An empty string could make impossible to select the post or other actions

**Comment by yakky** [comment]
Replaces #443

File: djangocms_blog/models.py (hunk 1 above)
Comment (sidneijp) [reply djangocms_blog/models.py]: I'll do the change and also edit the `tests.tests_models.ModelTest2.test_str_repr` to check this "no translation" case. Then pull request the `release/0.9.x` as you asked.

Comment (sidneijp) [reply djangocms_blog/models.py]: PR #451

Comment (yakky) [reply djangocms_blog/models.py]: Thanks, I'll close this

**Comment by yakky** [comment]
Replaced by #451

**Pull request closed without merging**